test/extensions/filters/http/rbac/rbac_filter_test.cc

```diff
@@ -129,6 +162,23 @@ TEST_F(RoleBasedAccessControlFilterTest, Path) {
       {":authority", "host"},
   };
   EXPECT_EQ(Http::FilterHeadersStatus::Continue, filter_.decodeHeaders(headers, false));
+
+  headers = Http::TestRequestHeaderMapImpl{
```

Comment:
what's the purpose of this and the next line?

Reply:
I added an extra test to Path, I thought it could maybe use a verify false test? I'll remove it. It's probably not necessary.

Reply:
isn't it duplicate with the above one?

Reply:
":path" is set to "prefix/suffix/next" for the first one and "/suffix#seg?param=value" for the second one.